tests/test_temporal_with_activity.py

```diff
@@ -124,3 +124,16 @@ def test_activity_on_entity_edit_duplicate_activity(self, session):
             with t.clock_tick(create_activity):
                 t.column = 4567
             session.commit()
+
+    def test_expire_clears_current_tick_and_activity(self, session):
+        create_activity = models.Activity(description='Create temp')
+        session.add(create_activity)
+        t = models.FirstTemporalWithActivity(column=1234,
+                                             activity=create_activity)
+        session.add(t)
+
+        assert getattr(t, '__temporal_current_tick', None)
```

Comment:
I wonder if we want to assert that it's an instance of `ClockTick` (or whatever it actually is) and has properties we expect?